docs/how_to/04-templates.rst

```diff
@@ -2,27 +2,169 @@
 How to work with templates
 ##########################
 
-Application can reuse cms templates by mixing cms template tags and normal django
-templating language.
+django CMS uses Django's template system to manage the layout of the pages. While
```

Comment:
"While" needs removing because what comes next is the title for the next section